setup.py

```diff
@@ -22,10 +24,10 @@ def readme():
         'Programming Language :: Python :: 3',
     ],
     install_requires=[
-        'click',
-        'beautifulsoup4',
-        'Markdown',
-        'readchar',
+        'click==7.1.2',
+        'beautifulsoup4==4.8.2',
+        'Markdown==3.2.1',
+        'readchar==2.0.1',
```

Comment:
pinning packages is good practice, avoid a lot of issues due to incorrect versions being used, eg anki 2.1.34 vs 2.1.26 in https://github.com/lervag/apy/pull/39/files#diff-f4c265073ea4274a80eca85440875e56R3)
You can use services like the [renovatebot github app](https://renovate.whitesourcesoftware.com/) to check packages upgrades and create PRs for you.

Reply:
Seems like good advice. Feel free to teach me about stuff like renovatebot; I've never heard of it. The most "demanding" dependency is Anki itself, as you've probably understood.